packages/env/lib/config/parse-config.js

```diff
@@ -293,6 +300,10 @@ async function parseRootConfig( configFile, rawConfig, options ) {
 		checkPort( configFile, `testsPort`, rawConfig.testsPort );
 		parsedConfig.testsPort = rawConfig.testsPort;
 	}
+	if ( rawConfig.afterSetup !== undefined ) {
```

Comment:
Since `parseRootConfig` runs on the default config as well, this would be another reason to leave the default value undefined.